content/departments/engineering/teams/integrations/integrations-support-rotation.md

```diff
@@ -0,0 +1,58 @@
+# Integrations support rotation
+
+> Note: This processes is currently in trial mode, meaning we will re-evaulate the value it brings on Nov 21 (~2 months after the introduction).
+
+The [Ingrations team](./index.md) has a dedicated member on a weekly support rotation. This document outlines the responsibilities of this role and how to execute the tasks. In a nutshell, these are the responsibilities of the engineer:
+
+- Respond and triage incoming inquiries from other teams and the support channels
+
+## Expectations
+
+The engineer on rotation is expected to comply with these responsibilities only **_within their usual working hours_**. There is no on-call, 24x7-watch expectation on these tasks. The async nature of Sourcegraph allows us to properly support the organization without having to be in front of the computer all day.
+
+You are not expected to provide all answers to inquiries or review all PRs. You are expected to be the initial responder and decide whether to tackle the work right away or add it for later planning. As a general rule of thumb, any work that will take longer than half a day should become an issue in the team's [GitHub board](https://github.com/orgs/sourcegraph/projects/213/views/18). This also includes work that takes less than half a day but needs to be transitioned to another engineer.
```

Comment:
>You are expected to be the initial responder and decide whether to tackle the work right away or add it for later planning

I would be more specific here and say that they are responsible for triaging the issue and assigning a priority based on [this](https://handbook.sourcegraph.com/departments/ce-support/support/process/support-prioritization/#priority-definitions) or something similar.

Reply:
I'd also say that one of the responsibilities is to bring any support issues that need to be solved in the milestone to our planning meeting. This will help in case we get a bunch of P1 issues that need support.

Reply:
> I'd also say that one of the responsibilities is to bring any support issues that need to be solved in the milestone to our planning meeting. This will help in case we get a bunch of P1 issues that need support.

I’m not sure I understand. If something is P1 it should probably be worked on ASAP. The next planning meeting might be two weeks away and the next sprint start could also be 2 weeks away. There's also a mismatch between a one week support rotation and a two week sprint cycle so that every other week we just pass on the work to the person next in the rotation?

IMO p1 issues should be surfaced in our teams internal channel so that we have a chance to react to it sooner and evaluate wether we should work on it in the current milestone or in the next one async.

Reply:
Perhaps P1 was a poor example as the SLA would mean you'd have to work on it straight away and I agree with your points. 

The scenario I was trying to cover was an influx of issues (not P1) that must be sorted within the our sprint in order to hit our SLAs. In that case, it may require assistance or just an FYI to the team that they're going to be busy.

Reply:
Gotcha, yeah this makes sense. I’m thinking on a good way to write this down, I don't want to add this as a responsibility to the person that is on the support schedule while we have our planning meetings since that will not rotate probably if we have an even number of team members. Perhaps we can rely on the handover messages in Slack and make it so that we look at the last two weeks of handover messages while we are in our planning meetings?

Reply:
I’m still unsure about how to communicate this. To me, raising something to the team is a safety net that we should encourage but, if possible, avoid since it will increase the workload of everyone. Don't you think the thing that you have in mind is already handled by the "finally" section in:

> Communicating any leftover tasks/asks/things to keep in mind for whoever is up next. This is best done the day before the rotation transitions via an async recap update posted in `#integrations-internal`.